lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -651,7 +651,7 @@ void SessionStateEnforcer::prepare_response_to_access(
   rsp->mutable_session_ambr()->set_max_bandwidth_dl(
       config.rat_specific_context.m5gsm_session_context()
           .default_ambr()
-          .max_bandwidth_ul());
+          .max_bandwidth_dl());

```

Comment:
Should we add a unit test for `prepare_response_to_access` to test these conversions?

Reply:
Hi Marie, 
UT code for this part is already present, please refer here: https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp?L764

https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp?L788:41

https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/session_manager/test/Matchers.h?L301

Reply:
Ah I see. Can we change the uplink and the downlink values? It seems like that test alone was not able to catch this typo.

Reply:
Modified uplink and downlink values to different values in UT code as you suggested.

Reply:
Hey @khansiddiquekc , thanks for changing tha tests

For future occasion, it is good to keep in mind that if a bug is fuound that means a unit test failed to catch the issue. So a bug like that should always trigger a change on test file  We need to make sure the unit test is robust enough.